typescript/api/services/I18nEntriesService.ts

```diff
@@ -86,9 +91,22 @@ export module Services {
                 await this.setBundle(defaultBrand, lng, ns, json, undefined, { splitToEntries: true, overwriteEntries: false });
                 this.logger.debug(`Seeded bundle ${defaultBrand.id}:${lng}:${ns}`);
               } else {
-                // Incremental: add any new keys found in defaults into entries; do not touch bundle data
-                await this.syncEntriesFromBundle({ branding: defaultBrand, locale: lng, namespace: ns, id: existing.id, data: json }, false);
-                this.logger.debug(`Synced new keys for ${defaultBrand.id}:${lng}:${ns}`);
+                // Incremental: merge defaults into existing bundle data
```

Comment:
(in unchanged code above) Could the load of the existing bundle be after the check for the file?
`const existing = await this.getBundle(defaultBrand, lng, ns);`
Is there a point to loading the existing data when there's no file?